pyswervedrive/swervemodule.py

```diff
@@ -171,7 +171,7 @@ def set_velocity(self, vx, vy):
         # if we have a really low velocity, don't do anything. This is to
         # prevent stuff like joystick whipping back and changing the module
         # azimuth
-        if velocity < 0.1:
+        if velocity == 0:
```

Comment:
We will have to bump up the deadzone on the joystick quite a bit. Could you do that please?

Reply:
I don't really see any point in tuning the joystick response until we start driver training/consult with the drive team, or at the very least until we have real hardware access.